### PR TITLE
recharter

### PR DESCRIPTION
**Pull request by murillo128**

### CODE REVIEW

File: charter/01-00.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The WISH working group is chartered to specify a simple, extensible,
 HTTPS-based signaling protocol to establish one-way WebRTC-based audiovisual
```
Comment (englishm) [inline charter/01-00.md]: This sentence would also need to be amended to describe more than a singular protocol.

**Review by steely-glint — CHANGES_REQUESTED** [review]
As I said at IETF 116 - I like the simplicity of these changes, but I think that removing the final paragraph goes too far. We risk becoming a general SIP-over-HTTP working group.

How about :

While there may be other problems that the proposed mechanism may solve or nearly solve, such as general purpose bidirectional realtime communication (telephony, video conferencing etc), adding explicit protocol support for those use cases is not in scope for the WISH working group.

**Comment by murillo128** [comment]
good to me

File: charter/01-00.md (hunk 1 above)
Comment (murillo128) [reply charter/01-00.md]: changed `protocol` by `set of protocols`, wdyt?

**Pull request merged**